hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java

```diff
@@ -111,8 +112,9 @@
      */
     public BulkUploadHandler(final TargetBulkUpdateWindowLayout targetBulkUpdateWindowLayout,
             final TargetManagement targetManagement, final ManagementUIState managementUIState,
-            final DeploymentManagement deploymentManagement, final I18N i18n) {
+            final DeploymentManagement deploymentManagement, final I18N i18n, final UI uiInstance) {
```

Comment:
Yes, because otherwise you would not see the notification in the appropriate browser window if you are working with more than one browser in the same session.